### PR TITLE
ui/TextFormField

### PR DESCRIPTION
**Pull request by AbdullahAhmedSoomro**
- TextField Created
- Some changes pending

### CODE REVIEW

File: lib/presentation/shared/text_field.dart [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            borderSide: BorderSide.none,
+            borderRadius: BorderRadius.circular(8.0),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class JournalTextField extends StatelessWidget {
```
Comment (hashirshoaeb) [inline lib/presentation/shared/text_field.dart:35]: make generic text field, should be compatible with single or multiline

**Comment by hashirshoaeb** [comment]
is it complete?

**Comment by AbdullahAhmedSoomro** [comment]
Nope

**Comment by AbdullahAhmedSoomro** [comment]
Stuck into changing the height of the container according to text field height


https://user-images.githubusercontent.com/50518325/216892181-3f387b40-590a-4c93-9a96-2c0f18108a94.mp4

**Pull request merged**